### ISSUE TITLE
Test failing for ingredients

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#1361]
```
#   Failed test at t/ingredients.t line 184.
#     Structures begin differing at:
#          $got->{ingredients_hierarchy}[0] = 'en:flour'
#     $expected->{ingredients_hierarchy}[0] = 'fr:farine'

```

**Comment by hangy** [comment]
Was temporarily fixed, but broken again by the taxonmy update in 07073400182a8e7a9aba32e0d62f0bfe14189db0: https://travis-ci.org/openfoodfacts/openfoodfacts-server/builds/447106751#L3450
```
#   Failed test at t/ingredients.t line 785.
#     Structures begin differing at:
#          $got->{ingredients_tags}[19] = 'en:sunflower'
#     $expected->{ingredients_tags}[19] = 'en:lecithins'
```

**Comment by hangy** [comment]
Fixed in cea7e9f07390a346c213655aacb41c5ddb2c562a
